### PR TITLE
Add Intern Job Page

### PR DESCRIPTION
**Pull request by Luke-Sikina**
- Add new page for intern
- Remove former members from team page
- Add link to intern page on positions page
- Update gitignore to include hidden files generated from
running `jekyll serve`

### CODE REVIEW

File: _pages/cbioportal-intern.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* Comfortable with basic git commands
+* Available during normal (EDT timezone) working hours for most of the summer
+* Able to work remotely
+
+
+The candidate must demonstrate excellent communication skills, outstanding personal initiative and the ability to work effectively as part of a team. Ability to meet deadlines and efficiently multitask is a must. 
+
+# How to Apply
+
+Please email your resume and cover letter to `cbioportal at ds.dfci.harvard.edu`.
```
Comment (Luke-Sikina) [inline _pages/cbioportal-intern.markdown:39]: @pieterlukasse Is this how we're doing submissions?

**Comment by pieterlukasse** [comment]
Approved 👍

**Comment by ecerami** [comment]
@pieterlukasse @Luke-Sikina this is now deployed at:  https://ds.dfci.harvard.edu/knowledge-systems/cbioportal-intern/

**Comment by ecerami** [comment]
@pieterlukasse @Luke-Sikina I also tweeted here:  https://twitter.com/ecerami/status/1381646081465929733

**Pull request merged**